src/views/fram-work/components/left/left-two.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div class="content">
+        <h2>原材料消耗</h2>
+        <div ref="echarts" :style="{height: maxHeight * 0.9 + 'px'}">
+  
+        </div>
+    </div>
+  </template>
```

Comment:
```suggestion
</template>
```

还有这个文件下面的缩进可以格式化一下
下同